packit/api.py

```diff
@@ -604,6 +609,8 @@ def run_copr_build(
                         inside (defaults to something long and ugly)
         :param chroots: a list of COPR chroots (targets) e.g. fedora-rawhide-x86_64
         :param owner: defaults to username from copr config file
+        :param description: description of the copr build
+        :param instructions: instructions of the copr build
```

Comment:
Actually, I think it's the `Description` of the project, not the build. And the instructions are also the `Installation Instructions` for the project. See for example [this](https://copr.fedorainfracloud.org/coprs/jpopelka/libtiff3/)